### ISSUE TITLE
Minification?

### ISSUE THREAD

**Issue by Kurren123** [phenax/esbuild-plugin-elm#16]
Should this plugin try to minify the resulting js if we are in production mode? [This post](https://discourse.elm-lang.org/t/what-i-ve-learned-about-minifying-elm-code/7632) gives this example:

```js
const fs = require("fs");
const esbuild = require("esbuild");

const pureFuncs = [ "F2", "F3", "F4", "F5", "F6", "F7", "F8", "F9", "A2", "A3", "A4", "A5", "A6", "A7", "A8", "A9"];

const elmCode = fs.readFileSync("elm.js", "utf8");

const result = esbuild.transformSync(elmCode, {
  minify: true,
  pure: pureFuncs,
  target: "es5",
});
```

**Comment by phenax** [comment]
Since, this is just a plugin, I believe minification should be explicitly handled in the esbuild config instead of being a part of a plugin. But I do think specifying the `pure` property from within the plugin if the `minify` flag is true, is a great idea.
Let me know if you're up for creating a pr for this. If not, I can pick this up some time in the next couple of days.

**Comment by Kurren123** [comment]
Thanks. Unfortunately I'm at max capacity for the next few months, but if you do get time to implement this whenever you are available it would make the minification process much smoother for beginners. 

Thanks again

**Comment by phenax** [comment]
Released in [0.0.10](https://github.com/phenax/esbuild-plugin-elm/releases/tag/v0.0.10)
